met-api/src/met_api/config.py

```diff
@@ -306,17 +318,25 @@ class DevConfig(Config):  # pylint: disable=too-few-public-methods
 class TestConfig(TestKeyConfig, Config):  # pylint: disable=too-few-public-methods
     """
     The configuration used when running the test suite.
+
     Extends TestKeyConfig, which contains some large constant keys that are used
     in the tests. It is stored in a separate file to avoid clutter.
     TestKeyConfig, in turn, extends the default Config class from above.
     """
 
     def __init__(self) -> None:
+        """
+        Initialize the object.
+
+        This method is called when an object is created. It sets up the initial
```

Comment:
Comments are always appreciated! Especially to add more context to an existing file